.github/workflows/release.yml

```diff
@@ -18,6 +18,17 @@ jobs:
       - env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN
+      - name: Temporarily disable "include administrators" branch protection
+        uses: benjefferies/branch-protection-bot@1.0.7
+        with:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The docs list this as `ACCESS_TOKEN` and describe it as a PAT. I'm not sure it's the same as what I'm using. We'll see.